cmd/ipfs-cluster-ctl/formatters.go

```diff
@@ -208,6 +212,23 @@ func textFormatPrintMetric(obj *api.Metric) {
 	fmt.Printf("%s: %s | Expire: %s\n", peer.IDB58Encode(obj.Peer), obj.Value, date)
 }
 
+func textFormatPrintRepoGC(obj *api.RepoGC) {
```

Comment:
If we add `--local` support we will have to change this. for PinInfo the `textFormatPrintPInfo` method creates a GlobalPinInfo object and calls the formatter for that one, so that output is always consistent. You could copy that (and also the peer sorting and could also carry peer names in the object).

Reply:
added support for `--local`

Instead of converting an object into a global object in formatter, shouldn't this happen at the API level? That would make API output consistent as well.

Reply:
Added peer name and sorted objects by peers.

But if this is about delivering same results(Is it?), with formatting we have done only for cli. 
Results in APIs are not sorted. Should we use array instead of map, if aim is to achieve consistent output?

Reply:
Have added support for local, peers are now sorted and peername is added